collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.odk.collect.android.utilities;
+
+import android.widget.Toast;
+
+import org.odk.collect.android.application.Collect;
+
+/**
```

Comment:
I can’t speak for the project, but I find these noisy and wish Studio wouldn’t put them in by default. Git tells us what we need to know.